docs/components/receive.md

```diff
@@ -101,34 +101,39 @@ The configuration file follows a few standards:
 
 All the configuration for the remote write endpoint of Receive is contained in the `write` key. Inside it there are 3 subsections:
 
-- `global`: limits and/or gates that are applied considering all the requests.
+- `global`: limits, gates and/or options that are applied considering all the requests.
 - `default`: the default values for limits in case a given tenant doesn't have any specified.
 - `tenants`: the limits for a given tenant.
 
 From the example configuration below, it's understood that:
 
 1. This Receive instance has a max concurrency of 30.
-2. This Receive instance has some default request limits that apply of all tenants, **unless** a given tenant has their own limits (i.e. the `acme` tenant and partially for the `ajax` tenant).
-3. Tenant `acme` has no request limits.
-4. Tenant `ajax` has a request series limit of 50000 and samples limit of 500. Their request size bytes limit is inherited from the default, 1024 bytes.
+2. This Receive instance has head series limiting enabled as it has `meta_monitoring_.*` options in `global`.
+3. This Receive instance has some default request limits as well as head series limits that apply of all tenants, **unless** a given tenant has their own limits (i.e. the `acme` tenant and partially for the `ajax` tenant).
```

Comment:
This numbering makes me feel like we're talking about difference instances, how about something like `Receive instance with configuration below...`.

Alternatively, we could drop the description or just add in-line comments in the YAML excerpt below, since the behavior of default vs. tenant-specific limits is clearly described above.